custom/enikshay/integrations/nikshay/repeaters.py

```diff
@@ -216,7 +216,7 @@ def allowed_to_forward(self, episode_case):
 
         episode_case_properties = episode_case.dynamic_case_properties()
         return (
-            episode_case_properties.get('nikshay_registered', 'false') == 'false' and
+            episode_case_properties.get('private_nikshay_registered', 'false') == 'false' and
```

Comment:
I don't think so, since none of them have succeeded yet. This would basically just make "dublicate entry" in Nikshay :P